program/src/processor.rs

```diff
@@ -1,25 +1,168 @@
 //! Program state processor
 
+use crate::state::Backpointer;
+use crate::{
+    _get_wrapped_mint_authority_signer_seeds, _get_wrapped_mint_backpointer_address_signer_seeds,
+    _get_wrapped_mint_signer_seeds, get_wrapped_mint_address_with_seed,
+    get_wrapped_mint_authority_with_seed, get_wrapped_mint_backpointer_address_with_seed,
+};
+use solana_program::account_info::next_account_info;
+use solana_program::program::invoke_signed;
+use solana_program::program_error::ProgramError;
+use solana_program::program_pack::Pack;
+use solana_program::rent::Rent;
+use solana_program::sysvar::Sysvar;
+use solana_program::{msg, system_instruction};
+use spl_token_2022::instruction::initialize_mint2;
 use {
     crate::instruction::TokenWrapInstruction,
     solana_program::{account_info::AccountInfo, entrypoint::ProgramResult, pubkey::Pubkey},
-    spl_token_2022::instruction::decode_instruction_type,
 };
 
+/// Processes [`CreateMint`](enum.TokenWrapInstruction.html) instruction.
+pub fn process_create_mint(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    idempotent: bool,
+) -> ProgramResult {
+    let account_info_iter = &mut accounts.iter();
+
+    let wrapped_mint_account = next_account_info(account_info_iter)?;
+    let wrapped_backpointer_account = next_account_info(account_info_iter)?;
+    let unwrapped_mint_account = next_account_info(account_info_iter)?;
+    let _system_program_account = next_account_info(account_info_iter)?;
```

Comment:
Conceptually not sure why a non-used account needs to be passed when it's not used in the code. However, tests indeed throw without it.